services/core-api/app/config.py

```diff
@@ -157,6 +159,13 @@ def JWT_ROLE_CALLBACK(jwt_dict):
     EMAIL_ENABLED = os.environ.get('EMAIL_ENABLED', False)
     EMAIL_RECIPIENT_OVERRIDE = os.environ.get('EMAIL_RECIPIENT_OVERRIDE')
 
+    #Templates
+    TEMPLATE_FOLDER_BASE = os.environ.get('TEMPLATE_FOLDER_BASE', '/app/app/templates')
+
+    TEMPLATE_FOLDER_IRT = os.environ.get('TEMPLATE_FOLDER_IRT', f'{TEMPLATE_FOLDER_BASE}/projects/')
+    TEMPLATE_IRT = os.environ.get('TEMPLATE_IRT',
```

Comment:
Should the default here be "IRT_Template.xlsx" like I see the file named in the templates folder?